pom.xml

```diff
@@ -1387,6 +1388,11 @@ under the License.
       </dependency>
       <dependency>
         <groupId>org.webjars</groupId>
+        <artifactId>google-diff-match-patch</artifactId>
```

Comment:
As you are introducing a new dependency, did you check its license?
Is it this one? https://code.google.com/archive/p/google-diff-match-patch/

If so, it's AL 2.0, so it should be good.

In general, any new dependency must comply with this ASF policy: https://www.apache.org/legal/resolved.html#category-a